seed/challenges/02-data-visualization-certification/react-projects.json

```diff
@@ -27,6 +27,20 @@
       "tests": [],
       "type": "zipline",
       "challengeType": 3,
+      "nameRu": "Создайте предпросмотрщик облегченного языка разметки Markdown",
+      "descriptionRu": [
+        "<strong>Задание:</strong> Создайте приложение <a href='http://codepen.io' target='_blank'>CodePen.io</a>, функционал которого схож с этим: <a href='http://codepen.io/FreeCodeCamp/full/obYYqW' target='_blank'>http://codepen.io/FreeCodeCamp/full/obYYqW</a>.",
+        "<strong>Правило #1:</strong> Не подсматривайте в код приложения-примера. Разберитесь самостоятельно.",
+        "<strong>Правило #2:</strong> Приложение должно удовлетворять нижеприведённым <a href='http://en.wikipedia.org/wiki/User_story' target='_blank'>пользовательским историям</a>. Используйте любые библиотеки или API, которые потребуются. Придайте ему свой личный стиль.",
+        "<strong>Правило #3:</strong> Для создания этого проекта вы должны использовать Sass и React.",
+        "<strong>Пользовательская История:</strong> Я могу использовать облегчённый язык разметки в стиле GitHub в текстовом поле.",
+        "<strong>Пользовательская История:</strong> Я могу видеть предварительный просмотр вывода моей облегченной разметки по мере ввода текста.",
+        "<strong>Подсказка:</strong> Вам не нужно интерпретировать облегчённую разметку самостоятельно - вы можете импортировать библиотеку Marked для этого: <a href='https://cdnjs.com/libraries/marked'>https://cdnjs.com/libraries/marked</a>",
+        "<strong>Заметка:</strong> Если вы хотите использовать синтаксис React JSX, вам понадобится задействовать 'Babel' в качестве препроцессора.",
+        "Не забывайте пользоваться методом <a href='//github.com/FreeCodeCamp/freecodecamp/wiki/How-to-get-help-when-you-get-stuck' target='_blank'>Читай-Ищи-Спрашивай</a>, если застрянете.",
+        "Когда закончите, нажмите кнопку \"I've completed this challenge\" и укажите ссылку на вашу работу на CodePen.",
+        "Вы можете получить отзыв о вашем проекте от коллег, поделившись ссылкой на неё в нашем <a href='//gitter.im/freecodecamp/codereview' target='_blank'>Чате для Рассмотрения Кода</a>. Также вы можете поделиться ею через Twitter и в Кемпинге вашего города (на Facebook)."
```

Comment:
> ...`в Кемпинге вашего города (на Facebook).` sounds not so good. `на странице Facebook вашего города` is a better option.

Isn't `on the Facebook page of your city` misleading? E.g. if one is located in Minneapolis, then should link be posted here: https://www.facebook.com/cityofminneapolis/ ? Shouldn't Free Code Camp be mentioned somehow, what do you think?